exchanges/okx/okx.go

```diff
@@ -3509,9 +3518,9 @@ func (ok *Okx) GetSignalBotEventHistory(ctx context.Context, algoID string, afte
 
 // ****************************************** Recurring Buy *****************************************
 
-// PlaceRecurringBuyOrder recurring buy is a strategy for investing a fixed amount in crypto at fixed intervals.
-// An appropriate recurring approach in volatile markets allows you to buy crypto at lower costs. Learn more
-// The API endpoints of Recurring buy require authentication.
+// PlaceRecurringBuyOrder recurringorder.Buy.Lower()is a strategy for investing a fixed amount in crypto at order.Sell.Lower()s.
+// An appropriate recurring approach in volatile markets allows you toorder.Buy.Lower()crypto at lower costs. order.Sell.Lower()re
+// The API endpoints of Recurringorder.Buy.Lower()order.Sell.Lower()n.
```

Comment:
Please revert all these comment changes across this file for `order.Buy.Lower()` and `order.Sell.Lower()`